develop/parachain-devs/runtime-development/build-deterministic-runtime.md

```diff
@@ -0,0 +1,135 @@
+- [**`subwasm`**](https://github.com/chevdor/subwasm){target=\_blank} - provides command-line options for working with the metadata and Wasm runtime built using srtool. The `subwasm` program is also used internally to perform tasks in the `srtool` image
+- [**`srtool-actions`**](https://github.com/chevdor/srtool-actions){target=\_blank} - provides GitHub actions to integrate builds produced using the `srtool` image with your GitHub CI/CD pipelines
+- [**`srtool-app`**](https://gitlab.com/chevdor/srtool-app){target=\_blank} - provides a simple graphical user interface for building the runtime using the `srtool` Docker image
+
+## Prepare the Environment
+
+It is recommended to install the `srtool-cli` program to work with the Docker image using a simple command-line interface.
+
+To prepare the environment:
+
+1. Open a terminal shell
+
+2. Verify that Docker is installed by running the following command:
+
+    ```bash
+    docker --version
+    ```
+
+    If Docker is installed, the command displays version information:
+
+    --8<-- 'code/develop/parachain-devs/runtime-development/build-deterministic-runtime/deterministic-runtime-1.html'
+
+3. Install the `srtool` command-line interface by running the following command:
+
+    ```bash
+    cargo install --git https://github.com/chevdor/srtool-cli
+    ```
+
+4. View usage information for the `srtool` command-line interface by running the following command:
+
+    ```bash
+    srtool help
+    ```
+
+5. Download the latest `srtool` Docker image by running the following command:
+
+    ```bash
+    srtool pull
+    ```
+
+## Start a Deterministic Build
+
+After preparing the environment, the Wasm runtime can be compiled using the `srtool` Docker image.
+
+To build the runtime, use the following command:
+
+```bash
+srtool build --app --package node-template-runtime --runtime-dir runtime
+```
+
+The following options are provided:
+
+- The name specified for the `--package` should be the name defined in the `Cargo.toml` file for the runtime
+- The path specified for the `--runtime-dir` should be the path to the `Cargo.toml` file for the runtime
+- If the `Cargo.toml` file for the runtime is located in a `runtime` subdirectory, for example, `runtime/kusama`,  the `--runtime-dir` parameter can be omitted
+
+## Use srtool in GitHub Actions
+
+To add a GitHub workflow for building the runtime:
+
+1. Create a `.github/workflows` directory in the chain's directory
+2. In the `.github/workflows` directory, click **Add file**, then select **Create new file**
+3. Copy the sample GitHub action from `basic.yml` example in the [srtools-actions](https://github.com/chevdor/srtool-actions){target=\_blank} repository and paste it into the file you created in the previous step
+
+    ??? interface "`basic.yml`"
+
+        ```yml
```

Comment:
We could pull this src file directly from github. We should just pin the version in the `variables.yaml` file